File: prime.lua
#!/usr/bin/env lua
function isPrime (x)
    -- Here will be a for loop
    if math.sqrt(x) % 1 == 0 then
        return false
    end
    for i = 2, math.sqrt(x) do
        if x % i == 0 then
            return false
        end
    end
    return true;
end

primes = {}
i = 0
while (#primes < tonumber(arg[1])) do
    if isPrime(i) then
        table.insert(primes, i)
    end
    i = i + 1
end
for key, value in pairs(primes) do
    print(value)
end

File: prime.py
#!/usr/bin/env python
from __future__ import (absolute_import, division,
                        print_function, unicode_literals)


def is_prime(x):
    from math import sqrt, ceil
    if sqrt(x).is_integer():
        return False
    for i in range(2, int(ceil(sqrt(x)))):
        if x % i == 0:
            return False
    return True

if __name__ == '__main__':
    from sys import argv
    primes = []
    i = 0
    while len(primes) < int(argv[1]):
        if is_prime(i):
            primes.append(i)
        i += 1
    print(primes)
